codestodians/food-basket-front-end
# [Bug: #154052106] Adjust Input Text width

#### What does this PR do?
- It prevents the Input text from going out of border

#### Description of Task to be completed?
- When typing on the Text field, the input text should not go out of border.

#### How should this be manually tested?
- Type on any of the text fields on the login/Signup forms, the input text should not go out of border

#### What are the relevant pivotal tracker stories?
[#154052106](https://www.pivotaltracker.com/story/show/154052106)
